README.rst

```diff
@@ -24,6 +24,10 @@ The last stable release is available on github and can be installed with ``pip``
     $ git clone https://github.com/rqlite/pyrqlite.git
     $ pip install ./pyrqlite
 
+Often they are released on PyPI too, able to be installed via the package named ``rqlite``::
+
+    $ pip install rqlite
```

Comment:
No, what I meant was that I have registered the name `rqlite` up on PyPI, but I have not pushed any packages up there.